pkg/yttlibrary/overlay/insert_annotation.go

```diff
@@ -4,11 +4,11 @@
 package overlay
 
 import (
+	"carvel.dev/ytt/pkg/template"
+	tplcore "carvel.dev/ytt/pkg/template/core"
+	"carvel.dev/ytt/pkg/yamltemplate"
```

Comment:
I think these should be below "fmt", probably adding a new line between "fmt" and "github.com/k14s/starlark-go/starlark" should resolve it.